index.html

```diff
@@ -120,12 +125,15 @@ <h6>HTML</h6>
     </section>
     <!-- about myself section -->
     <section class="about-myself">
-        <h1 class="primary-text primary-text-about-myself">About me</h1>
+        <h2 class="primary-text primary-text-about-myself">About me</h2>
         <p class="supporting-text-about-me"> I can help you build a product , feature
             or website Look through some of my work and experience!
             If you like what you see and have a project you need coded,
             don’t hestiate to contact me.</p>
-        <a href="#" class="button button-about-me">Get My Resume</a>
+        <div class="about-me-illustration-btn">
+            <img src="Images/about-me-illustration.svg" alt="illustration">
+            <a href="#" class="button button-about-me">Get My Resume</a>
+        </div>
```

Comment:
- [ ] There is an issue with this button because it's aligned to the right and this is because of the `<img>` tag, and this issue is present in the desktop and the mobile version. I would suggest moving this image (since its being used as background) to the CSS file, after removing the `<img>` tag, the button will be centered :smiley:

  Mobile version (before):
  ![image](https://user-images.githubusercontent.com/81522435/144485547-d715d0ab-48df-497c-8e5a-9e2ecf701196.png)

  Desktop version (before):
  ![image](https://user-images.githubusercontent.com/81522435/144485597-954259bf-9792-4e82-8f06-f0795bce1144.png)

  After removing the `<img>` tag:
  ![image](https://user-images.githubusercontent.com/81522435/144485730-4993249b-146b-4122-94ca-a508b35c89eb.png)

Reply:
Thank you for pointing this out, and for taking the time to implement it after the removal of the `<img>` tag!